tests/e2e/constants/BASE_TEST_CONSTANTS.ts

```diff
@@ -130,6 +132,16 @@ export const BASE_TEST_CONSTANTS: {
 	 */
 	DELETE_WORKSPACE_ON_FAILED_TEST: process.env.DELETE_WORKSPACE_ON_FAILED_TEST === 'true',
 
+	/**
+	 * stop and remove workspace if a test is successful.
+	 */
+	DELETE_WORKSPACE_ON_SUCCESSFUL_TEST: process.env.DELETE_WORKSPACE_ON_SUCCESSFUL_TEST !== 'false',
+
+	/**
+	 * restart existing workspace.
```

Comment:
let's add the comment that , `false by default` to make it clear

Reply:
Fixed

Reply:
thanks